demos/classification_demo/grid_mat.hpp

```diff
@@ -57,32 +60,26 @@ class GridMat {
         testMessageSize = cv::getTextSize(testMessage, fontType, fontScale, thickness, &baseline);
     }
 
-    void textUpdate(double avgFPS, double avgLatency, double accuracy,
-                    bool isFpsTest, bool showAccuracy,
+    void textUpdate(PerformanceMetrics::TimePoint lastRequestStartTime,
+                    cv::Mat& frame,
```

Comment:
What's the purpose of this parameter? You always pass `outImg` as the value anyway.

Reply:
Good note